packages/components/table-v2/src/use-table.ts

```diff
@@ -139,7 +139,7 @@ function useTable(props: TableV2Props) {
 
     if (
       unref(lastRenderedRowIndex) >= 0 &&
-      _totalHeight === scrollTop + unref(mainTableHeight) - unref(headerHeight)
+      _totalHeight.toFixed(2) >= (scrollTop + unref(mainTableHeight) - unref(headerHeight)).toFixed(2)
```

Comment:
Perhaps we can consider two values ​​equal if the difference is less than 0.001. 🤔

Reply:
> Perhaps we can consider two values ​​equal if the difference is less than 0.001. 🤔
But there is a situation where the two are never equal due to accuracy issues, for example, the result of 19.9+10.2===30.1 is false

Reply:
> Perhaps we can consider two values ​​equal if the difference is less than 0.001. 🤔

Perhaps there is a better solution, but I think it is unreasonable to judge based on congruence.